tests/Application/webpack.config.js

```diff
@@ -1,22 +1,59 @@
 const path = require('path');
 const Encore = require('@symfony/webpack-encore');
-const syliusBundles = path.resolve(__dirname, 'vendor/sylius/sylius/src/Sylius/Bundle/');
-const uiBundleScripts = path.resolve(syliusBundles, 'UiBundle/Resources/private/js/');
-const uiBundleResources = path.resolve(syliusBundles, 'UiBundle/Resources/private/');
-// Admin config
-Encore
-    .setOutputPath('public/build/admin/')
-    .setPublicPath('/build/admin')
-    .addEntry('admin-entry', '../../src/Resources/assets/js/main.js')
-    .disableSingleRuntimeChunk()
-    .cleanupOutputBeforeBuild()
-    .enableSourceMaps(!Encore.isProduction())
-    .enableVersioning(Encore.isProduction())
-    .enableSassLoader();
+// const pluginConfig = require('../../webpack.config');
```

Comment:
remove it if it's not used

Reply:
it can be used, Patryk will describe it in readme/documentation, but its a part of configuration. I just leaved it as it is for now.

Reply:
We have 2 ways of configuring this webpack, one of these ways is commented :D